### PR TITLE
New rule for motorola.de

### PR DESCRIPTION
**Pull request by KommX**
One could merge this rule into the existing "Motorola.xml" but I haven't done it because there are missing tests.

### CODE REVIEW

**Comment by TheNavigat** [comment]
I believe that it's cleaner to add the tests to Motorola.xml and then add the .de domain to the hosts.

**Comment by KommX** [comment]
Ok, I think I managed to do it.

**Comment by TheNavigat** [comment]
LGTM

**Comment by TheNavigat** [comment]
LGTM.

File: src/chrome/content/rules/Motorola.xml [hunk 1]
```diff
@@ -63,23 +63,39 @@
...
+		to="https://www.motorolasolutions.com/" />
+	
 	<rule from="^http://motorola\.com/"
 		to="https://www.motorola.com/" />
+		
+	<rule from="^http://www\.motorola\.de/"
+		to="https://www.motorola.de/" />
 
 	<rule from="^http://(globalrepairtools|mediacenter|membership|sdc|www)\.motorola\.com/"
 		to="https://$1.motorola.com/" />
```
Comment (youdly) [inline src/chrome/content/rules/Motorola.xml]: For cleaner rule, you can simply line 97-101 to
`<rule from="^http:"
    to="https:" />`

Comment (KommX) [inline src/chrome/content/rules/Motorola.xml]: This doesn't seem to be possible.
After your recommended change 'test.sh' complains:

ERROR src/chrome/content/rules/Motorola.xml: Not enough tests (0 vs 1) for <Rule from '^http://cdn\.motorola-2\.com/' to 'https://cdn.motorola-2.com/'>
ERROR src/chrome/content/rules/Motorola.xml: Not enough tests (0 vs 1) for <Rule from '^http://mediacenter\.motorolasolutions\.com/' to 'https://mediacenter.motorolasolutions.com/'>

Comment (youdly) [inline src/chrome/content/rules/Motorola.xml]: Remove line 103-104 & 109-110.

Comment (KommX) [inline src/chrome/content/rules/Motorola.xml]: Thank you for your suggestions. I now committed them.

**Pull request merged**